README.md

```diff
@@ -1 +1,20 @@
-# React-calculator
\ No newline at end of file
+Milestone 3 React Calculator 
```

Comment:
- [ ] [OPTIONAL] I'd strongly encourage you to update your README file to the template linked by my colegues, the reason for that is that you'll most likelly have to do that in the future in career/interview prep and that a better README file can make recruiters more prone to contact you about jobs and opportunities.